lib/ngsiHandlers.js

```diff
@@ -159,10 +159,8 @@ function ngsiUpdateHandler(id, type, service, subservice, attributes, callback)
         async.map(attributes, async.apply(writeAttribute, ngsiDevice, lwm2mDevice), innerCallback);
```

Comment:
CNR entry missing (similar to the ones included for other IOTAs)

Reply:
Fixed in https://github.com/telefonicaid/lightweightm2m-iotagent/pull/253/commits/f88f776e4631adb3eda41b991cdecbddf12860be